rotkehlchen/externalapis/opensea.py

```diff
@@ -315,12 +366,30 @@ def gather_account_collections(self, account: ChecksumEvmAddress) -> None:
                 name='floor price',
                 location='opensea',
             )
+
+            if (
+                len(payment_tokens := collection.get('payment_tokens', [])) == 0 or
+                stats_result['total']['floor_price_symbol'] in ('', 'ETH')
```

Comment:
Both in your new code and some lines above this function does not seem to handle `KeyError` properly for `stats_result`. Please do so and if you decide to handle it at the caller side, note in the docstring that the function raises a KeyError